### ISSUE TITLE
Discussion about Data storage

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#149]
## Description
Have a discussion about how and where to store datasets, regarding size and performance.

## Acceptance criteria
* Convenient way of storing the data 
* Fast data accession
* Documentation

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Corgam** [comment]
Discussed during the last sprints. Closing.
